src/cmd-buildextend-installer

```diff
@@ -249,42 +274,46 @@ def generate_iso():
         run_verbose(['/usr/lib/coreos-assembler/osmet-pack',
                      img_metal4k, '4096', tmp_osmet4k, img_metal4k_checksum,
                      'fast' if args.fast else 'normal'])
-        if not args.no_pxe:
-            cp_reflink(tmp_osmet, os.path.join(tmpinitrd_pxe_or_rootfs, os.path.basename(tmp_osmet)))
-            cp_reflink(tmp_osmet4k, os.path.join(tmpinitrd_pxe_or_rootfs, os.path.basename(tmp_osmet4k)))
 
     # Generate root squashfs
-    tmp_squashfs = None
     if is_live:
         print(f'Compressing squashfs with {squashfs_compression}')
-        tmp_squashfs = os.path.join(tmpisoroot, 'root.squashfs')
+        # Name must be exactly "root.squashfs" because the 20live dracut
+        # module makes assumptions about the length of the name in
+        # sysroot.mount
+        tmp_squashfs = os.path.join(tmpinitrd_rootfs, 'root.squashfs')
         run_verbose(['/usr/lib/coreos-assembler/gf-mksquashfs',
                     img_metal, tmp_squashfs, squashfs_compression])
-        if not args.no_pxe:
-            cp_reflink(tmp_squashfs, os.path.join(tmpinitrd_pxe_or_rootfs, 'root.squashfs'))
 
-    # Generate initramfses
+    # Generate rootfs image
+    if is_live:
+        iso_rootfs = os.path.join(tmpisoimagespxe, rootfs_img)
+        extend_initrd(iso_rootfs, tmpinitrd_rootfs, compress=False)
```

Comment:
Might be worth a short comment saying why we leave it uncompressed. Your commit message is great. Maybe something similar here (or maybe slightly more brief). Sometimes when code gets moved around the "git blame" loses its effectiveness.